File: prime.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    int number = get_int("number:");
    
    int divided = 0;
    for(int i = 2; i <= sqrt(number); i++)
    {
        if(number%i == 0)
        {
            divided = 1;
            break;
        }
    }
    
    if(divided == 0 && number > 1)
    {
        printf("Prime\n");
    }
    else
    {
        printf("Not Prime\n");
    }
}

File: prime.py
from math import sqrt
from sys import exit

x = int(input("Enter a number: "))
if x < 0:
    exit("Negative numbers cannot be prime numbers.")

divided = False

for i in range(2, int(sqrt(x)) + 1):
    if x == 2:
        break
    if x % i == 0:
        divided = True
    
if divided == False and x > 1:
    print("Prime")
else:
    print("Not prime")
    
    
